src/game/server/NextBot/NextBotBehavior.h

```diff
@@ -565,8 +568,14 @@ class Behavior : public INextBotEventResponder, public IContextualQuery
 	}
 
 
+#ifdef NEO
+public:
+	Action< Actor > *m_action;
+private:
+#else
 private:
 	Action< Actor > *m_action;
+#endif
```

Comment:
Nit: Would it be neater to declare `friend class CNEOBotBehavior;` so we didn't need to publicly expose this?